contacts.js

```diff
@@ -0,0 +1,55 @@
+const fs = require("fs").promises;
+const path = require("path");
+
+class Contacts {
+    constructor() {
+        this.fileContactsPath = path.resolve(__dirname, "db", "contacts.json");
```

Comment:
make it as constant

        this.FILE_CONTACTS_PATH = path.resolve(__dirname, "db", "contacts.json");